qa/test_template.sh

```diff
@@ -40,7 +40,7 @@ epilog=${epilog-:}
 for i in `seq 0 $last_config_index`;
 do
     echo "Test run $i"
-    for variant in $(seq 0 ${#prolog[@]}); do
+    for variant in $(seq 0 $((${#prolog[@]}-1))); do
```

Comment:
Is this evaluating the number of elements in prolog? If you don't mind could you add a comment explaining what the `$((${#prolog[@]}-1)))` does? `#prolog[@]` is number of elements, maybe is it worth a separate variable for better understanding and readability.

Reply:
Done